testing/e2e/main_test.go

```diff
@@ -19,7 +19,6 @@ var (
 	agentPortCounter                     int64 = 20000
 	agentPath, backendPath, sensuctlPath string
 	binDir                               = filepath.Join("..", "..", "bin")
-	toolsDir                             = filepath.Join(binDir, "tools")
```

Comment:
Curious why this is in the PR, seems unrelated?

Reply:
Oh my bad, related to https://github.com/sensu/sensu-go/pull/1713, forgot to merge master back.